lib/vsc/install/shared_setup.py

```diff
@@ -1353,9 +1358,6 @@ def action_target(package, *args, **kwargs):
     """
     This main is the setup.py for vsc-install
     """
-    install_requires = [
```

Comment:
the problem is, currently clusterbuildrpm is doing the translation from python- prefixed to not python- prefixed (I guess?), so it works on our buildhosts with clusterbuildrpm, but it doesn't work locally on my machine, because when I run `python setup.py bdist_rpm` I get
```
error: Failed build dependencies:
	setuptools is needed by vsc-install-0.10.6-1.noarch
error: command 'rpmbuild' failed with exit status 1
```

so we should maybe move these translation thingies from clusterbuildrpm to vsc-install to fix this?